compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SymbolEnter.java

```diff
@@ -644,6 +646,13 @@ private SemType resolveTypeDesc(BLangBuiltInRefTypeNode td, Env semtypeEnv) {
         }
     }
 
+    private SemType resolveTypeDesc(BLangTableTypeNode td, Env semtypeEnv, Map<String, BLangNode> mod, int depth,
+                                    BLangTypeDefinition typeDefinition) {
+        SemType memberType = resolveTypeDesc(semtypeEnv, mod, (BLangTypeDefinition) td.constraint.defn, depth,
```

Comment:
Can we break just after `=` sign to get all in same line, or else I think we can try to break just after the open parenthesis.